microservices-connector/config/manifests/tei.yaml

```diff
@@ -110,12 +110,14 @@ spec:
               port: http
             initialDelaySeconds: 5
             periodSeconds: 5
+            timeoutSeconds: 2
```

Comment:
The manifest files are  under `microservices-connector/config/manifests` directory is generated by `helm-charts/update_manifests.sh` automatically. So if you want to change the default probe settings, please change it in corresponding component's helm chart, i.e. `helm-chart/common/tei/values.yaml`. Otherwise, these values will be easily overwritten.

Reply:
@lianhao Yes.  Using `cpu-values.yaml` in ChatQnA papers over the (CPU) issues with current Helm charts, but a proper fix would be needed for generating the GMC manifests.  However, that is *not* HPA specific problem, and requires more discussion[1][2] so IMHO it was clearly out of scope for this PR (although scaling makes these problems _much_ more visible).

[1] What's IMHO needed to fix things for probe timings:
* Common components' value files include different probe timing sections for CPU and for accelerators
* Their deployment templates select one based on `.Values.accelDevice` value (empty for CPU)
* All `<device>-values.yaml` files set appropriate `<subchart>.accelDevice` value (not just ChatQnA)
* GMC device variant manifests are generated for all relevant components, not just TGI

(I don't think probe timings would need to be fine-tuned based on which model is used.)

[2] What's IMHO needed to fix resource requests:
* Current sub-optimal component arguments are optimized, and resulting resource requirements are profiled, for all relevant models
  * For example on SPR, CPU TGI data type can be set to bfloat16, which **halves** its memory usage
  * Observability support + suitable Grafana dashboards will help with profiling
* Instead of subcomponent model & corresponding resources being specified in top-level chart, `helm install` command uses suitable model+resource+args file from given component, like this:
   * `-f common/tgi/gaudi/neural-chat-7b.yaml`
   * `-f common/teirerank/gaudi/bge-reranker-base.yaml`
   * `-f common/tei/cpu/bge-base.yaml`
   * `-f common/data-prep/gpu/values.yaml`
   * (These would provide values with subchart prefix/heading so they can be used from top-level charts)
* There could also be a global option for ignoring (CPU side) resource requests that can be used when things need to be re-profiled
* _GMC applies resource specs generated from these, when user changes model_

If there are combinations of above which are common between different top-level charts, I would suggest `Makefile` merging relevant ones to common files (e.g. `-f common/gaudi-model-defaults.yaml`), to avoid duplicating things that may need to be updated whenever args, model, or image versions get updated.

Reply:
@eero-t Thanks for the suggestion. I think we need further discussion about this. I created the draft PR for us to discuss